### DISCUSSION TITLE
Archive Entries

### DISCUSSION THREAD

**Discussion by x80486** [ente-io/ente#491] [Enhancements]
## Description

Would be great to have a way to "put aside" some entries that aren't quite ready yet to be removed, but either got superseded or retired by some others...or just old.

## Describe the Solution You Would Like

A section/folder (or something similar) to place the entries — like an `Archive` section/folder.

## Describe Alternatives You've Considered

Leaving them with the rest, changing some part of the description to make them different.

## Additional Context

N/A

**Comment by ua741** [comment]
We can now use tags as a way to archive entries.
